_scripts/instruction-widget/install.js

```diff
@@ -107,8 +110,8 @@ module.exports = function(context) {
       context.certonly = true;
     }
 
-    // Debian Jessie backports.
-    if (context.version == 8) {
+    // Jessie backports.
+    if ((context.devuan && context.version == 1) || context.version == 8) {
```

Comment:
Yes. Devuan is very similar to Debian. If you have Devuan testing or unstable, it's available in the official repos.